### PR TITLE
Admission Control Threat Model

### PR DESCRIPTION
**Pull request by raesene**
This is a PR for the admission control threat model work that we've been doing in SIG-Security Docs.

### CODE REVIEW

**Comment by raesene** [comment]
@savitharaghunathan does this one look ok to merge, or do you think we've got some changes to make? Just thinking that ideally it'd be good to get this merged before the blog post comes out, so we can link the threat model from the blog post :)

File: sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md [hunk 1]
```diff
@@ -0,0 +1,601 @@
+# Kubernetes Admission Control Threat Model
+
+
+[TOC]
```
Comment (savitharaghunathan) [inline sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md:4]: @raesene is this intended to be left empty?

**Comment by savitharaghunathan** [comment]
> @savitharaghunathan does this one look ok to merge, or do you think we've got some changes to make? Just thinking that ideally it'd be good to get this merged before the blog post comes out, so we can link the threat model from the blog post :)

I just have the TOC question. Other than that its `lgtm` from me. Thanks @raesene :)

**Comment by PushkarJ** [comment]
Does this PR address / fix issue #7 ?

**Comment by savitharaghunathan** [comment]
> Does this PR address / fix issue #7 ?

@PushkarJ yes :)

**Comment by PushkarJ** [comment]
/lgtm

(Adding this for K8s two person review guidance, as now @savitharaghunathan can approve when she thinks its ready)

@raesene Loving the use of deciduous, btw

File: sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md (hunk 1 above)
Comment (raesene) [reply sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md]: good point I think that was just a placeholder from the google docs --> markdown conversion so I removed it.

**Comment by raesene** [comment]
@savitharaghunathan cool thanks, I've removed that TOC link it was just an artifact of the conversion from gdocs.

@PushkarJ cool thanks for the review :)

**Comment by savitharaghunathan** [comment]
Thanks @raesene 😊

/lgtm
/approve 
/hold 
For @PushkarJ

**Pull request merged**